src/collections/components/CollectionListCard/CollectionListCard.tsx

```diff
@@ -21,7 +21,9 @@ interface CollectionListCardProps {
    * An object with everything collection-related in it.
    * Except for stories. We don't need them in the card.
    */
-  collection: Omit<Collection, 'stories'>;
+  collection: Omit<Collection, 'stories' | 'image' | 'partnership'> & {
+    image?: Omit<Image, 'src'> | null;
```

Comment:
`src` on the new `Type Image` is now deprecated.

I'm not fetching the `src` field under the `image` field on the `Collection` type in the queries/mutations. So I have to omit it from this prop type - won't be necessary once we've replaced all uses of `src` in this code base and can eventually remove the deprecated field from the schema.